packages/agoric-cli/src/init.js

```diff
@@ -2,7 +2,6 @@ import chalk from 'chalk';
 import { makePspawn } from './helpers.js';
 
 /// <reference types="@endo/captp/src/types.js" />
-/// <reference types="@agoric/swingset-vat/exported.js" />
```

Comment:
removing that file seems like a breaking change

for example, consider a dapp that imports it.

_comment belongs on the deleted file, but I don't see how to do that._

Reply:
true. in practice we know all the code that relies on it:
https://github.com/search?q=org%3AAgoric+swingset-vat%2Fexported&type=code

https://github.com/Agoric/dapp-pegasus/blob/f3b8f0400e9921a7497b08385d2b3aa609829182/api/deploy.js#L9

that looks ok to me to drop but if you want a BREAKING CHANGE commit I'm game. the package hasn't hit version 1 yet so it won't be a major version bump

Reply:
In order to accommodate 3rd party developers, yes, I think a BREAKING CHANGE commit is worthwhile.

For example:

https://github.com/Kryha/KREAd/blob/c5c5d9e4b7cc370b0adfadc407ee7199c2064c98/agoric/contract/src/kreadKit.js#L43

Reply:
Ok I'll amend the commit message before I merge. I do wish we didn't have to modify commit history in order to note such things

Reply:
oh. I misread this PR as deleting `@agoric/zoe/exported.js`.